### PR TITLE
Kubernetes v1.22 support

### PR DESCRIPTION
**Pull request by invidian**

### CODE REVIEW

**Comment by roobre** [comment]
Can we rebase this and check if the new pixie operator _installs_ (even if it doesn't run) on 1.22? That way we could keep the chart in the dependencies list.

**Comment by roobre** [comment]
Do we foresee for Kube-certgen to take much time to release an official version of the image? If this is not the case, we can upload the temporary image to the `newrelic` dockerhub repo to avoid having personal repos in `master`.

File: charts/nri-metadata-injection/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic metadata injection webhook.
 name: nri-metadata-injection
-version: 1.5.4
-appVersion: 1.5.0
+version: 1.5.5
+appVersion: v1.5.0-20-gc893468
```
Comment (gsanchezgavier) [inline charts/nri-metadata-injection/Chart.yaml:5]: just released [v1.6.0](https://github.com/newrelic/k8s-metadata-injection/releases/tag/v1.6.0)

**Review by gsanchezgavier — COMMENTED** [review]
Thanks, 
Could you bump the  released integrations 
- k8s-metadata-injection v1.6.0
- newrelic-infra-operator v0.5.0
- kube-events v1.6.0
Including the dependencies bump for k8s 1.22

**Comment by invidian** [comment]
Thanks @gsanchezgavier! I updated the PR to include new releases.

File: charts/nri-metadata-injection/values.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 injectOnlyLabeledNamespaces: false
 
 image:
-  repository: newrelic/k8s-metadata-injection
+  repository: invidian/k8s-metadata-injection
```
[the remark this answers is not in the document]
Comment (invidian) [reply charts/nri-metadata-injection/values.yaml]: Yes, my bad, fixed.

File: charts/nri-metadata-injection/values.yaml [hunk 3]
```diff
@@ -16,8 +16,8 @@ image:
     # - name: regsecret
 
 jobImage:
-  repository: jettech/kube-webhook-certgen
-  tag: v1.5.0
+  repository: invidian/kube-webhook-certgen
```
[the remark this answers is not in the document]
Comment (invidian) [reply charts/nri-metadata-injection/values.yaml]: Yes, fixed, thanks for pointing out!

**Comment by roobre** [comment]
Taking over for the 1.22 release, thanks for working on this @invidian!

**Comment by roobre** [comment]
Closing as changes were included in #527

**Pull request closed without merging**